app/logic/Calendar/Invitation.ts

```diff
@@ -1,20 +1,24 @@
 export type iCalMethod = "CANCEL" | "REQUEST" | "REPLY";
 
 /**
- * For an inbox item that represents a scheduling message, the type of message:
- * Accepted/Tentative/Declined responses, invitations, or cancellations.
+ * For an inbox item that represents a scheduling message, the type of message.
  */
-export enum Scheduling {
+export enum InvitationMessage {
   None = 0,
-  Accepted = 1,
-  Tentative = 2,
-  Declined = 3,
-  Request = 4,
-  Cancellation = 5,
+  Invitation = 4, // METHOD:REQUEST, an incoming invitation
+  CancelledEvent = 5, // METHOD:CANCEL, a follow-up cancellation
+  ParticipantReply = 6, // METHOD:REPLY, a reply to your outgoing invitation
```

Comment:
Make these JSDoc, please. Like so:
```suggestion
  /** Creates an incoming invitation.
   * iCal METHOD: REQUEST */
  Invitation = 4,
  /** In incoming invitation is being nullified.
   * Only valid for incoming invitations.
   * iCal METHOD: CANCEL */
  CancelledEvent = 5,
  /** A reply to your outgoing invitation.
   * Only valid for outgoing invitations.
   * iCal METHOD: REPLY */
  ParticipantReply = 6,
```

Reply:
(iCal does not allow extra whitespace in its values)

Reply:
You can change `METHOD` to lowercase `method`.